sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactsFilterForm.java

```diff
@@ -347,6 +353,18 @@ protected void applyDependenciesOnFieldChange(String propertyId, Property.ValueC
 			populateSurveillanceOfficersForDistrict(district);
 			break;
 		}
+		case ContactIndexDto.DISEASE: {
+			Disease disease = (Disease) event.getProperty().getValue();
+			ComboBox diseaseVariantField = getField(ContactCriteria.DISEASE_VARIANT);
+			if (disease != null) {
+				List<DiseaseVariantReferenceDto> diseaseVariants = FacadeProvider.getDiseaseVariantFacade().getAllByDisease(disease);
+				FieldHelper.updateItems(diseaseVariantField, diseaseVariants);
+				diseaseVariantField.setEnabled(true);
```

Comment:
The `diseaseVariantField` should be set to `enabled` only if the `diseaseVariants` is not empty.

Reply:
The other dependent fields in the form are enabled even though there are no "child" entries. Also, enabling results in the field description to be shown. Otherwise, it is just a blank, disabled field, which does not give any feedback to the user.

Reply:
I understand, however this behavior would be more consistent with the requirement of the ticket:
> The disease variant filter is only enabled, when a disease with variants is selected
and with the way Disease Variant filter is working for cases. see `de.symeda.sormas.ui.caze.CaseFilterForm:555`